# DP: suggest installation of python-tk package on failing _tkinter import

--- a/Lib/tkinter/__init__.py
+++ b/Lib/tkinter/__init__.py
@@ -34,7 +34,10 @@ import enum
 import sys
 import types
 
-import _tkinter # If this fails your Python may not be configured for Tk
+try:
+    import _tkinter
+except ImportError as msg:
+    raise ImportError(str(msg) + ', please install the python3-tk package')
 TclError = _tkinter.TclError
 from tkinter.constants import *
 import re
